### PR TITLE
numeric patching for degree/radian conversions

### PR DESCRIPTION
**Pull request by rs-ehyde**

### CODE REVIEW

File: calculation.rb [hunk 1]
```diff
@@ -1,39 +1,58 @@
...
+    # degrees to radians, where +self+ is the degree
+    self * Math::PI / 180
   end
 
-  boat_calc = Calculation.new
+  def to_degrees
+    # radians to degrees, where +self+ is the radian
+    self * 180 / Math::PI
+  end
+end
```
Comment (rs-ehyde) [inline calculation.rb:42]: Adding Numeric methods for to_radians and to_degrees

File: calculation.rb [hunk 2]
```diff
@@ -1,39 +1,58 @@
...
-      maybe_apparent_wind_angle * 180 / Math::PI
+  attr_accessor :variance, :eta
+
+  def initialize
+    @variance = 0
+    @eta = 0.01
+  end
+
+  def determine_apparent_wind_angle(true_wind, wind_velocity)
+    true_wind_radians = true_wind.to_radians
```
Comment (rs-ehyde) [inline calculation.rb:10]: used here

File: calculation.rb [hunk 3]
```diff
@@ -1,39 +1,58 @@
...
 
-apparent_wind_angle = boat_calc.determine_apparent_wind_angle(true_wind_angle, wind_velocity)
+puts "apparent wind angle"
+apparent_wind_radians = calculator.determine_apparent_wind_angle(1, 10)
+puts apparent_wind_radians.to_degrees
 
-  puts 10 * ( Math.sin(1.5708 - apparent_wind_angle) / Math.sin(apparent_wind_angle) ).abs() 
\ No newline at end of file
+puts "boat speed"
+puts 10 * ((Math.sin(1.to_radians - apparent_wind_radians) / apparent_wind_radians).to_degrees)
```
Comment (rs-ehyde) [inline calculation.rb:58]: and here

**Review by Shaka-n — APPROVED** [review]
Hard to believe there aren't already methods on Numeric for radian/degree conversion.

**Pull request merged**